docs/tutorials/model-registry.txt

```diff
@@ -0,0 +1,245 @@
+.. _organizing-models:
+
+Organizing Models in the Model Registry
+=======================================
+
+In this tutorial, we focus on organization of checkpoints via the Determined
```

Comment:
Can we include some discussion here about _why_ you'd want to organize models in a model registry, some of the use-cases this feature is designed to support, etc.? (Or if that information appears in a different doc that is fine.)

Reply:
Will do! This is going to be the only doc apart from the autogenerated reference material so I will add an introduction covering use cases.